dcos-log/api/v2/handlers.go

```diff
@@ -280,13 +295,21 @@ func redirectURL(id *nodeutil.CanonicalTaskID, file, RawQuery string) (string, e
 
 	// take the last element
 	taskID := id.ContainerIDs[len(id.ContainerIDs)-1]
-	taskLogURL := fmt.Sprintf("%s/%s/logs/v2/task/frameworks/%s/executors/%s/runs/%s/", prefix, id.AgentID,
+	taskLogURL := fmt.Sprintf("%s/%s/logs/v2/task/frameworks/%s/executors/%s/runs/%s", prefix, id.AgentID,
 		id.FrameworkID, executorID, taskID)
 
 	if isPod {
-		taskLogURL += fmt.Sprintf("/tasks/%s/%s", id.ID, file)
+		taskLogURL += fmt.Sprintf("/tasks/%s", id.ID)
```

Comment:
might be worth using `path.Join` here and further down where we are building paths for the `taskLogURL`...